third_party/move/move-model/src/metadata.rs

```diff
@@ -153,8 +155,8 @@ impl CompilerVersion {
     }
 
     /// The latest stable compiler version.
-    pub fn latest_stable() -> Self {
-        CompilerVersion::from_str(LATEST_STABLE_COMPILER_VERSION).expect("valid version")
+    pub const fn latest_stable() -> Self {
+        LATEST_STABLE_COMPILER_VERSION_VALUE
```

Comment:
Not sure why we need this change. Now, to upgrade the latest stable language version, we need to change both `LATEST_STABLE_LANGUAGE_VERSION` and `LATEST_STABLE_LANGUAGE_VERSION_VALUE`, and make sure to keep them in sync.

Instead, in the previous version, you would only need to change the `LATEST_STABLE_LANGUAGE_VERSION`, and if it is set to the wrong string, tests would fails.

Reply:
@brmataptos Just to make sure you saw this comment. The PR is ready to go once this is addressed.

Reply:
We need a `const` value to use with the `config` structs in `tests.rs`.  While we could use a string, that's needlessly indirect.  I've The old definition of `latest_stable()` cannot be declared as `const`, due to the non-const nature of trait functions like `from_str()` (apparently `const traits` are currently "unstable", but also there may be types for which `from_str()` wouldn't be constant).  In this case, defining a custom `to_str()` for `CompilerVersion` and `LanguageVersion` allows us to derive `LATEST_STABLE_COMPILER_VERSION` from `LATEST_STABLE_COMPILER_VERSION_VALUE` and have a `const` value we can store in the config.